packages/beacon-node/src/api/rest/swaggerUI.ts

```diff
@@ -0,0 +1,81 @@
+import {FastifyInstance} from "fastify";
+import {BeaconRestApiServerOpts} from "./index.js";
+
+export async function registerSwaggerUIRoutes(
+  server: FastifyInstance,
+  opts: BeaconRestApiServerOpts,
+  version = ""
+): Promise<void> {
+  await server.register(await import("@fastify/swagger"), {
+    openapi: {
+      info: {
+        title: "Lodestar API",
+        description: "API specification for the Lodestar beacon node",
+        version,
+        contact: {
+          name: "Lodestar Github",
+          url: "https://github.com/chainsafe/lodestar",
+        },
+      },
+      externalDocs: {
+        url: "https://chainsafe.github.io/lodestar",
+        description: "Lodestar documentation",
+      },
+      tags: opts.api.map((namespace) => ({name: namespace})),
+    },
+  });
+  await server.register(await import("@fastify/swagger-ui"), {
+    theme: {
+      title: "Lodestar API",
+      favicon: await getFavicon(),
+    },
+    logo: await getLogo(),
+  });
+}
+
+/**
+ * Fallback-friendly function to get an asset
+ */
+async function getAsset(name: string): Promise<Buffer | undefined> {
+  try {
+    const path = await import("node:path");
+    const fs = await import("node:fs/promises");
+    const url = await import("node:url");
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
+    return await fs.readFile(path.join(__dirname, "../../../../../assets/", name));
```

Comment:
Just noticed we don't copy `assets` in our docker build
https://github.com/ChainSafe/lodestar/blob/4fd3d4d3020872c8bbeefa80dc08989b42200e41/.dockerignore#L8

I think we can just include them, size impact it relatively low
```
1.2M    assets
288.1M  node_modules
```